omod/src/main/webapp/fragments/patientHeader.gsp

```diff
@@ -119,27 +119,18 @@
                     <i class="toggle-icon icon-caret-up small"></i>
                 </a>
             </span>
+
+            <div class="firstLineFragments">
+                <% firstLineFragments.each { %>
+                    ${ ui.includeFragment(it.extensionParams.provider, it.extensionParams.fragment, [patient: config.patient])}
+                <% } %>
+            </div>
+
             <div class="hidden" id="contactInfoContent" class="contact-info-content">
                 ${ ui.includeFragment("coreapps", "patientdashboard/contactInfoInline", [ patient: config.patient, contextModel: appContextModel ]) }
             </div>
         </h1>
-        <% if (config.activeVisit) { %>
-            <% def visit = config.activeVisit.visit %>
 
-            <div class="active-visit-started-at-message">
-                ${ui.message("coreapps.patientHeader.activeVisit.at", config.activeVisitStartDatetime)}
```

Comment:
The page was previously designed so that the patient identifiers divs (`<div class="identifiers">`) were almost at the end of the page HTML, but with a `float: right` CSS option to display them at the top right of the patient header.
But as soon as the active visit and admission status tags were getting longer (or we were adding more tags), the patient ids were moving at the bottom right of the patient header:
![patient-ids-at-bottom-right](https://cloud.githubusercontent.com/assets/10692811/20665980/31105730-b562-11e6-9ba1-d72096ccb16b.png)

That is the reason why I inserted the 'secondLineFragments' extension points **after** the `<div class="identifiers">` and added the CSS declaration `.secondLineFragments {clear: both}` to it. This way, it has a proper dedicated div section in the patient header, with the full width available.

Hope it makes sense